src/control/Control.Layers.js

```diff
@@ -113,6 +113,27 @@ L.Control.Layers = L.Control.extend({
 		return (this._map) ? this._update() : this;
 	},
 
+	// @method expand(): undefined
+	// Expand the control container if collapsed.
+	expand: function () {
+		L.DomUtil.addClass(this._container, 'leaflet-control-layers-expanded');
+		this._form.style.height = null;
+		var acceptableHeight = this._map.getSize().y - (this._container.offsetTop + 50);
+		if (acceptableHeight < this._form.clientHeight) {
+			L.DomUtil.addClass(this._form, 'leaflet-control-layers-scrollbar');
+			this._form.style.height = acceptableHeight + 'px';
+		} else {
+			L.DomUtil.removeClass(this._form, 'leaflet-control-layers-scrollbar');
+		}
+		this._checkDisabledLayers();
+	},
+
+	// @method collapse(): undefined

```

Comment:
Maybe return `this`, like most of our public methods which would otherwise be `void` do?